src/test/java/com/purbon/kafka/topology/PrincipalManagerTest.java

```diff
@@ -35,6 +35,8 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Properties;
+import java.util.Set;
+import jersey.repackaged.com.google.common.collect.Sets;
```

Comment:
is this import really required?

Reply:
we can do this by first building a collection and then pass it to the hashset constructor, isn't?